clients/go-ethereum/geth.sh

```diff
@@ -137,6 +138,11 @@ if [ "$HIVE_MINER_EXTRA" != "" ]; then
 fi
 FLAGS="$FLAGS --miner.gasprice 16000000000"
 
+# Configure any les flags
+if [ "$HIVE_LIGHTSERVE" != "" ]; then
+  FLAGS="$FLAGS --light.serve $HIVE_LIGHTSERVE --light.nosyncserve "
```

Comment:
The `light.nosyncserve` flag is only available after 1.9.25. We should ban all geth client less than 1.9.25 to run as the les server.